src/main/java/com/twilio/oai/TwilioNodeGenerator.java

```diff
@@ -250,6 +252,16 @@ private void addDependent(final Map<String, Object> dependents, final String dep
         dependent.put("filename", dependentName);
     }
 
+    private CodegenModel resolveComplexType(CodegenModel item) {
+        for (CodegenProperty prop: item.vars) {
+            if (prop.complexType != null) {
+                prop.dataType = prop.baseName.equals("test_object") ? "object" :
+                        prop.baseName.equals("test_array_of_objects") ? "Array<object>" : prop.dataType;
```

Comment:
This is still too coupled with the naming in the test fixture. The condition needs to be based off some property that is not based on the name in the fixture.

Reply:
`prop.baseType` works for the object array, but there's no helpful metadata in the `test_object` property (its base type and openapi type are still marked as `TestResponseObjectTestObject`).

Reply:
Anything indicating it's complex?